modules/ROOT/pages/teams.adoc

```diff
@@ -0,0 +1,67 @@
+
+Starting with the “Everyone at <Your Organization>" team, which contains all users who join your organization either via an invitation or SSO, you can create specialized _child teams_ that have more specific permissions based on the responsibilities of their members.
+The Teams feature enables you to mimic the structure of your organization and seamlessly manage permissions and team membership as your organization grows.
+
+After your child teams are created, you can invite new users directly to the appropriate team using email invitations or using the xref:configure-teams#invite-users-with-teams[Teams feature interface].
+
+When you add a user to a team, you can also specify whether they should be a _team maintainer_ (a user who receives the permissions outlined in the team structure and manages team membership) or a _member_ (a user who receives the permissions outlined in the team structure). Outside of teams, you can still xref:users.adoc#grant-user-permissions[assign specific permissions to individual users] if they require permissions that should not extend to an entire team.
+
+Adopting a team structure in your organization also enables you to share assets more easily. In Anypoint Exchange, you can share an asset with an entire team rather than having to select each user individually.
+
+[NOTE]
+To manage teams permissions, you must have the Organization Administrator permission over the business group that contains resources that you want to give access over to a specific user or a team.
+You can modify any permissions for all teams in your organization only if you have the Organization Administrator permission over the root organization.
+
+== Permissions Structure
+
+When you are planning your organization’s team structure, consider the following:
+
+* Each member of your organization is a member of the “Everyone at <Your Organization> team, or the _root_ team.
+* Your team structure should reflect the structure of your organization so that you can plan your teams and configure permissions according to member needs. However, you can move teams as the needs of your organization change.
+* Teams names must be unique across your organization.
+* Your organization can have up to 2000 teams.
+* Your teams can be nested up to 10 levels, including the “Everyone at <Your Organization> team.
+* You can create teams without adding members.
+* Every child team inherits permissions from its parent team.
+
+When you configure permissions in your organization, you should give your organization’s main team, “Everyone at <Your Organization>, permissions that do not afford too many privileges. Since every member of your organization is a member of that team, you should only configure permissions that you want every user to have. The child teams you create under the top level team inherit the permissions of the parent team. You can give each child team more specialized permissions that reflects the team members’ responsibilities.
```

Comment:
Note use here of "your organization's main team" instead of "root" to refer to Everyone....

Reply:
GLOBAL: "since" > "because" if that is what you really mean (rather than "after"...which you should also replace "since with!)

Reply:
"only configure" > "configure only those permissions"; also, suggested removal of modal "you should" here:
```suggestion
When you configure permissions in your organization, give your organization’s main team, “Everyone at <Your Organization>, permissions that do not afford too many privileges. Because every member of your organization is a member of that team, configure only those permissions that you want every user to have. The child teams you create under the top-level team inherit the permissions of the parent team. You can give each child team more specialized permissions that reflects the team members’ responsibilities.
```

Reply:
I don't understand the child team-top-level team-parent team hierarchy in this sentence:

"The child teams you create under the top-level team inherit the permissions of the parent team."

What is the top-level team: root? Are we repeating that all teams inherit from root, or repeating that all child teams inherit from their parent?

Reply:
All child teams inherit permissions from their parent team(s).

Reply:
Which includes root.